Gemfile

```diff
@@ -3,7 +3,9 @@
 source 'https://rubygems.org'
 ruby '>= 2.3.0', '< 2.5.0'
 
-gem 'pkg-config'
```

Comment:
i'm concerned this will affect other things, can you tell me more about what's happening here/why?